datafusion/core/src/physical_plan/aggregates/row_hash.rs

```diff
@@ -120,6 +151,56 @@ use super::AggregateExec;
 /// hash table).
 ///
 /// [`group_values`]: Self::group_values
+///
+/// # Spilling
+///
+/// The sizes of group values and accumulators can become large. Before that causes out of memory,
+/// this hash aggregator outputs those data early for partial aggregation or spills to local disk
```

Comment:
Using `emit` to match the API name. More easy to read accordingly with code.
```suggestion
/// this hash aggregator emits those data early for partial aggregation or spills to local disk
```

Reply:
The way I distinguish between `output` and `emit` is
`output`: output from the aggregation exec. 
`emit`: reading out from the hash table, rather more internal.
Since the subject in this sentence is `hash aggregator`, I chose `output`